account_credit_control/models/credit_control_policy.py

```diff
@@ -29,6 +29,20 @@ class CreditControlPolicy(models.Model):
              " for the selected accounts",
     )
     active = fields.Boolean('Active', default=True)
+    default_on_partner = fields.Boolean(
+        string="Default Value On Partner"
+    )
+
+    @api.constrains('default_on_partner')
+    @api.multi
+    def _constrains_default_on_partner(self):
+        count = self.search_count([
+            ('default_on_partner', '=', True)
+        ])
```

Comment:
do we should take the do_nothing field into account.

Reply:
Same as https://github.com/OCA/account-financial-tools/pull/819/files#r329928573